src/Integration/Binding/ProjectBindingOperation.Writer.cs

```diff
@@ -45,22 +46,22 @@ internal partial class ProjectBindingOperation
         /// <param name="ruleSetFileName">The rule set file name</param>
         /// <param name="solutionRuleSet\">Full path of the parent solution-level SonarQube rule set</param>
         /// <returns>Full file path of the file that we expect to write to</returns>
-        internal /*for testing purposes*/ string QueueWriteProjectLevelRuleSet(string projectFullPath, string ruleSetFileName, ConfigFileInformation solutionRuleSet, string currentRuleSetPath)
+        internal /*for testing purposes*/ string QueueWriteProjectLevelRuleSet(string projectFullPath, string ruleSetFileName, IBindingConfigFile solutionRuleSet, string currentRuleSetPath)
```

Comment:
`solutionRuleSet` isn't a great name any more, although it could be changed when renaming `IBindingConfigFile`.